ports/windows/init.c

```diff
@@ -61,6 +62,7 @@ void init() {
     // https://msdn.microsoft.com/en-us/library/bb531344(v=vs.140).aspx
     _set_output_format(_TWO_DIGIT_EXPONENT);
 #endif
+    set_fmode_binary();
```

Comment:
Is this needed anymore?  In 9bdb82ef6b3147662ffc6b6435c6bd376c60e19e a call to this was added in mpy-cross itself.  And wouldn't the addition here change the behaviour of the windows port of MicroPython?

Reply:
Without this loading persistent code might fail because the binary .mpy file gets read while having CRLF replaced by LF etc; assuming the default mode for `open()` is text which does seem to be the case. So that indeed changes behaviour, as also briefly discussed in #2209, but I think this change is actually good: problem now is that writing `\n` to a file in uPy yields `\r\n` - which likely went unnoticed so far just because reading is symmetrical and does the opposite. I'll address this in the commit message.

mpy-cross doesn't call `init()` currently so is unaffected by this change, but it's probably better to change mpy-cross so it does the same initialization.